jira/jira_tools/tools/basic_funcs.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import requests
+from requests.exceptions import HTTPError
+
+ATLASSIAN_RESOURCES_URL = "https://api.atlassian.com/oauth/token/accessible-resources"
+ATLASSIAN_JIRA_API_URL = "https://api.atlassian.com/ex/jira"
+
+
+def get_jira_cloud_id() -> str:
+    token = os.getenv("JIRA_OAUTH_TOKEN", "")
+
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Accept": "application/json",
+    }
+    try:
+        response = requests.get(ATLASSIAN_RESOURCES_URL, headers=headers)
+        response.raise_for_status()
+        resources = response.json()
+        workspaces = [resources["name"] for resources in resources]
+        if len(resources) > 1:
+            print(f"WARNING: You have more than one workspace, available workspaces: {workspaces}. "
+                  f"The first one will be used...")
+        return resources[0]["id"]
+
+    except HTTPError as e:
+        print(f"Failed to get Jira server: {e}")
+        return ""
```

Comment:
I think you can raise an error, so we wont have to remember to check if it's empty